gap_analysis/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+gap_analysis
+==============
+
+basic models on the business analysis and time estimation.
```

Comment:
This module provides the basic models for business analysis and time estimation 
and is part of a set of modules.